test/extended/deployments/deployments.go

```diff
@@ -298,7 +298,14 @@ var _ = g.Describe("deploymentconfigs", func() {
 			_, name, err := createFixture(oc, envRefDeploymentFixture)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			out, err := oc.Run("deploy").Args("dc/"+name, "--latest", "--follow").Output()
+			_, err := oc.Run("rollout").Args("latest", name).Output()
```

Comment:
> wonder if it will work when the config does not have any triggers/was not deployed yet

As I said, [observedGeneration is updated](https://github.com/openshift/origin/blob/c3fc160805f0c06f656deda0524046a7d9963d9e/pkg/deploy/controller/deploymentconfig/controller.go#L82).